Makefile

```diff
@@ -1,6 +1,9 @@
 SRCS = main.c gpio.c fileutil.c logging.c
 OBJS = $(SRCS:.c=.o)
 LIBS = -lrt
+# _DEFAULT_SOURCE (glibc >= 2.19) for clock_gettime(),
+# strdup(), and daemon().
+CFLAGS = -Wall -Wpedantic -D_DEFAULT_SOURCE --std=c99
```

Comment:
I'm a little bit curious about the requirement for adding `-D_DEFAULT_SOURCE`.  That appears to be related to explicitly running with `--std=c99`, but it looks like I have a little reading in front of me to understand exactly what that is necessary.